### ISSUE TITLE
Release current master branch

### ISSUE THREAD

**Issue by unclechu** [webpack-contrib/coffee-loader#44]
I'm talking about 0ea4d6a7cc9ffc7c8f7a0c6eb3fd3fedea9943b8.

I was confused looking at this package source, because in the latest commit it uses `coffeescript` dependency but in last npm release (`0.7.3`) it uses `coffee-script`, so I had `1.3.3` version by `coffee-script` package name and when I had tried to build my app I got this error:

```
ERROR in ./resources/assets/script/main.coffee
Module build failed: SyntaxError: Unexpected token u in JSON at position 0
  at Object.parse (native)
  at Object.module.exports (/%absolute_path_hidden%/node_modules/coffee-loader/index.js:40:17)
```

Later I looked at line from stack-trace:
```
var map = JSON.parse(result.v3SourceMap);
```
So, skipping more details, my point is: in the last release coffee-script dependency is incorrect, not even package name, but version, `1.x`, so even `1.3.3` working incorrect.

Now I must declare dependency as:
```
"coffee-loader": "github:webpack-contrib/coffee-loader#0ea4d6a7cc9ffc7c8f7a0c6eb3fd3fedea9943b8",
```

**Comment by d3viant0ne** [comment]
@unclechu - Latest master has been released as `0.8.0`. Changelog with the permanent information for this release can be found [here](https://github.com/webpack-contrib/coffee-loader/blob/master/CHANGELOG.md#080-2017-08-23)

**Comment by unclechu** [comment]
thanks!
